Tasks/GruntV0/Tests/L0GruntCliNotSet.ts

```diff
@@ -1,30 +1,39 @@
-{
+import ma = require("azure-pipelines-task-lib/mock-answer");
+import tmrm = require("azure-pipelines-task-lib/mock-run");
+import path = require("path");
+
+let taskPath = path.join(__dirname, "..", "grunttask.js");
+let tr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
+
+tr.setInput("gruntFile", "gruntfile.js");
+tr.setInput("cwd", "fake/wd");
+
+// provide answers for task mock
+let a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
     "which": {
         "grunt": "/usr/local/bin/grunt",
         "npm": "/usr/local/bin/npm",
         "node": "/usr/local/bin/node",
         "istanbulWin": "/usr/local/bin/istanbul",
-        "istanbul": "/usr/local/bin/node_modules/istanbul/lib/cli.js"
+        "istanbul": "/usr/local/bin/node_modules/istanbul/lib/cli.js",
     },
     "exec": {
         "/usr/local/bin/grunt --gruntfile gruntfile.js": {
             "code": 0,
-            "stdout": "grunt output here"
+            "stdout": "grunt output here",
         },
         "/usr/local/bin/npm install istanbul": {
             "code": 0,
-            "stdout": "npm output here"
+            "stdout": "npm output here",
         },
         "/usr/local/bin/node ./node_modules/istanbul/lib/cli.js cover --report cobertura --report html -i ./**/build/src/*.js ./node_modules/mocha/bin/_mocha **/build/test/*.js": {
-            "code": 1,
+            "code": 0,
             "stdout": "istanbul output here",
-            "stderr": "istanbul failed with this output"
         },
         "/usr/local/bin/node ./node_modules/istanbul/lib/cli.js cover --report cobertura --report html -i .\\**\\build\\src\\*.js ./node_modules/mocha/bin/_mocha **/build/test/*.js": {
-            "code": 1,
+            "code": 0,
```

Comment:
Is it expected that it's succeded currently, but was failed before?

Reply:
There was no `L0GruntCliNotSet.ts` file before. It's just how github shows diffs
![image](https://user-images.githubusercontent.com/16704239/104718274-7b399880-56df-11eb-9c52-8a1651c019bd.png)